pkg/apis/core/v1/conversion.go

```diff
@@ -357,88 +357,6 @@ func Convert_v1_Secret_To_core_Secret(in *v1.Secret, out *core.Secret, s convers
 	return nil
 }
 
-func Convert_core_SecurityContext_To_v1_SecurityContext(in *core.SecurityContext, out *v1.SecurityContext, s conversion.Scope) error {
```

Comment:
compare with autogenerated function:
https://github.com/kubernetes/kubernetes/blob/c3e8a434ebef5478f1122bacc1da7423cb23678b/pkg/apis/core/v1/zz_generated.conversion.go#L6816-L6827

(the Convert_... functions being called were the autogenerated ones)